mmv1/third_party/terraform/services/networksecurity/resource_network_security_firewall_endpoint_association_test.go.erb

```diff
@@ -20,28 +21,91 @@ func TestAccNetworkSecurityFirewallEndpointAssociations_basic(t *testing.T) {
 	acctest.SkipIfVcr(t)
```

Comment:
This can be removed I believe. I looked into this resource in https://github.com/GoogleCloudPlatform/magic-modules/pull/10308 and though I didn't get the test to successfully pass I believe VCR should work for this test.

Reply:
@matheusaleixo-cit let's apply this change- it'll make our presubmits run the test.